package.json

```diff
@@ -125,7 +127,7 @@
     "test:smoke:components-js": "yarn --cwd ./packages/components-js test:smoke",
     "updateAlgolia": "yarn --cwd ./packages/storefront updateAlgolia",
     "clean": "yarn clean:node_modules && yarn clean:build",
-    "clean:node_modules": "rm -rf ./node_modules && rm -rf ./yarn-error.log && yarn clean:assets:node_modules && yarn clean:components:node_modules && yarn clean:components-angular:node_modules && yarn clean:components-js:node_modules && yarn clean:components-react:node_modules && yarn clean:components-vue:node_modules && yarn clean:email:node_modules && yarn clean:fonts:node_modules && yarn clean:icons:node_modules && yarn clean:marque:node_modules && yarn clean:meta-icons:node_modules && yarn clean:shared:node_modules && yarn clean:storefront:node_modules && yarn clean:utilities:node_modules && yarn clean:utilities-deprecated:node_modules",
+    "clean:node_modules": "rm -rf ./node_modules && rm -rf ./yarn-error.log && yarn clean:assets:node_modules && yarn clean:components:node_modules && yarn clean:components-angular:node_modules && yarn clean:components-js:node_modules && yarn clean:components-react:node_modules && yarn clean:components-vue:node_modules && yarn clean:crawler && yarn clean:email:node_modules && yarn clean:fonts:node_modules && yarn clean:icons:node_modules && yarn clean:marque:node_modules && yarn clean:meta-icons:node_modules && yarn clean:shared:node_modules && yarn clean:storefront:node_modules && yarn clean:utilities:node_modules && yarn clean:utilities-deprecated:node_modules",
```

Comment:
child command should be called `clean:crawler:node_modules` to match the rest.

Reply:
Changed it